enhancements/network/user-defined-network-segmentation.md

```diff
@@ -379,11 +382,283 @@ any user-defined primary network.
+
+| Field name     | Description                                                                                                                                                                                                                                                                                                                                                                                            | optional |
+|----------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|
+| Topology       | The topological configuration for the network. Must be one of `Layer2`, `layer3`, `Localnet`.                                                                                                                                                                                                                                                                                                          | No       |
+| Role           | Select the network role in the pod, either `Primary` or `Secondary`. Primary network support topologies `Layer2` and `Layer3` only.                                                                                                                                                                                                                                                                    | No       |
+| MTU            | The maximum transmission unit (MTU).<br/>The default is 1400.                                                                                                                                                                                                                                                                                                                                          | Yes      |
+| Subnets        | The subnet to use for the network across the cluster.<br/>E.g. `10.100.200.0/24`.<br/>IPv6 `2001:DBB::/64` and dual-stack `192.168.100.0/24`,`2001:DBB::/64` subnets are supported.<br/>When omitted, the logical switch implementing the network only provides layer 2 communication, and users must configure IP addresses.<br/>Port security only prevents MAC spoofing if the subnets are omitted. | Yes      |
+| ExcludeSubnets | List of CIDRs.<br/>IP addresses are removed from the assignable IP address pool and are never passed to the pods.                                                                                                                                                                                                                                                                                      | Yes      |
```

Comment:
Can you expand on the use case for this field? It's not obvious why I would want this, and what the effects of this would be

Reply:
most (all?) fields we have here are inherited from the NAD spec (UDN is kind of a better vesion of NAD with validations and status). So we tried to keep field names and value formats similar to the NAD, so that users familiar with NADs may easily switch their configs to the new CRD.
For specific use case of `ExcludeSubnets` I will ping @maiqueb :)